doc/additional_documentation.doc

```diff
@@ -0,0 +1,103 @@
+/******************************************************************************
+ *
+ *
+ *
+ * Copyright (C) 1997-2015 by Dimitri van Heesch.
+ *
+ * Permission to use, copy, modify, and distribute this software and its
+ * documentation under the terms of the GNU General Public License is hereby
+ * granted. No representations are made about the suitability of this software
+ * for any purpose. It is provided "as is" without express or implied warranty.
+ * See the GNU General Public License for more details.
+ *
+ * Documents produced by Doxygen are derivative works derived from the
+ * input used in their production; they are not affected by this license.
+ *
+ */
+/*! \page additional_documentation Additional Documentation
+\tableofcontents{html,latex}
+
+\section custom_pages Custom Pages
+
+\c doxygen can be also be used to create custom pages that are not part of the API of your library/program, 
+the purpose of these pages is to enrich your documentation with anything else that you think the user may find useful. (such as this page itself!)
+
+To create custom pages, use one of the supported the file formats: .dox, .txt, .md. 
+Inside such a file doxygen will treat it like any other source file, so one option could be to consider this file as large single doxygen comment.
+
+\c manual/index.dox
+\code
+/** \mainpage My Library Manual
+- Building
+- Basics
+- Examples
+*/
+\endcode
+
+You'll note that the `\mainpage` command was used, which tells doxygen to use this page as, well, the main page. For other pages, preface them with the `\page` command.
```

Comment:
It would be good to refere for the used commands in this chapter to refer to the documentation of that command )so for `\mainpage` this would be `\ref cmdmainpage "\\mainpage"`